### PR TITLE
Add custom event_time column as an option during baseline creation

### PR DESCRIPTION
**Pull request by sfc-gh-pkommini**
**Changes**
- Adds default options for custom temporal column named field to allow for easier baselining of tables in webUI
- Add logic to accept the custom temporal column for the table and uses it in the generated SQL
- Add additional files and dirs to `.gitignore`

**Testing**
- Ran through the workflow of adding a baseline entering the following on Dev WebUI and dev snowflake:
```
Base Table: `data.azure_collect_connection:recorded_at`
Window Size: 90
Sparcity Reduction: Leave to default
Group-by Columns: subscription_id
```
- Entering base table without `:` to delineate the time column. Result is it throws an error asking the user to enter a time column name as below:
image

- Entered non-existent time column and the query succeeds, however it fails later during the actual ingestion into the connection able.

### CODE REVIEW

**Review by sfc-gh-afedorov — CHANGES_REQUESTED** [review]
please test & describe testing in change description.

what does error look like if someone forgets `:` in the field?

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,6 @@ venv
 *.egg-info
 .vscode
 *.pydevproject
+dist/
```
[the remark this answers is not in the document]
Comment (sfc-gh-afedorov) [reply .gitignore:21]: gotcha -- looks like I mis-remembered, in node that's result of "Nuxt.js build", not standard

in Python, it's standard when you build something to distribute via, e.g. pypi? open to building towards this, as at least @sfc-gh-gbutzi has been interested in re-using pieces of the code we wrote in other scripts, but this seems like something we'd want to do in a separate design doc (with Greg's team giving feedback) and then in a separate PR.

[the remark this answers is not in the document]
Comment (sfc-gh-pkommini) [reply .gitignore]: Reverted the gitignore file.

**Comment by sfc-gh-pkommini** [comment]
Added screenshot of how the `:` not being present is handled.

**Pull request merged**